docs/assistant/connect-to-bedrock.asciidoc

```diff
@@ -0,0 +1,86 @@
+. From the **IAM** menu, select **Policies** → **Create policy**.
+. To provide the necessary permissions, paste the following JSON into the **Specify permissions** menu.
++
+```json
+{
+    "Version": "2012-10-17",
+    "Statement": [
+        {
+            "Sid": "VisualEditor0",
+            "Effect": "Allow",
+            "Action": [
+                "bedrock:InvokeModel",
+                "bedrock:InvokeModelWithResponseStream"
+            ],
+            "Resource": "*"
+        }
+    ]
+}
+```
++
+NOTE: These are the minimum required permissions. IAM policies with additional permissions are also supported.
++
+. Click **Next**. Name your policy.
+
+[discrete]
+=== Configure an IAM User
+
+. Return to the **IAM** menu. Select **Users** from the navigation menu, then click **Create User**. 
+. Name the user, then click **Next**.
+. Select **Attach policies directly**. 
+. In the **Permissions policies** field, search for the policy you created earlier, select it, and click **Next**.
+. Review the configuration then click **Create user**.
+
+[discrete]
+=== Create an access key
+
+. Return to the **IAM** menu. Select **Users** from the navigation menu. 
+. Search for the user you just created, and click its name.
+. Go to the **Security credentials** tab.
+. Under **Access keys**, click **Create access key**.
+. Select **Third-party service**, check the box under **Confirmation**, click **Next**, then click **Create access key**.
+. Click **Download .csv file** to download the key. Store it securely.
+
+[discrete]
+=== Enable model access
+
+. Search the AWS console for Amazon Bedrock. 
+. From the Amazon Bedrock page, click **Get started**.
+. Select **Model access** from the left navigation menu, then click **Manage model access**.
+. Check the boxes for **Claude** and **Claude Instant**, depending which model or models you plan to use.
+. Click **Save changes**.
+
+[discrete]
+== Configure Elastic AI Assistant
+
+. Log in to {kib}. 
+. Go to **Stack Management → Connectors → Create connector → Amazon Bedrock**. 
+. Name your connector. 
+. (Optional) Configure the Amazon Bedrock connector to use a different AWS region where Anthropic models are supported by editing the **URL** field, for example by changing `us-east-1` to `eu-central-1`.
+. (Optional) Add one of the following strings if you want to use a model other than the default:
+.. For Haiku: `anthropic.claude-3-haiku-20240307-v1:0`
+.. For Sonnet: `anthropic.claude-3-sonnet-20240229-v1:0`
+.. For Opus: `anthropic.claude-3-opus-20240229-v1:0`
+. Enter the **Access Key** and **Secret** that you generated earlier, then click **Save**.
+
+Your model is now configured. For more information on using Elastic AI Assistant, refer to <<security-assistant, AI Assistant>>.
+
+IMPORTANT: If you're using provisioned throughput, your ARN becomes the model ID, and the connector settings **URL** value must be https://www.urlencoder.org/[encoded] to work. 
```

Comment:
IMPORTANT: If you're using [provisioned throughput](https://docs.aws.amazon.com/bedrock/latest/userguide/prov-throughput.html), your ARN becomes the model ID, and the connector settings **URL** value must be [encoded](https://www.urlencoder.org/) to work.